backend/apps/beaver_etl/.env.example

```diff
@@ -0,0 +1,2 @@
+CHUNK_SIZE=10
+API_SECRET_TOKEN="4995e3277642496b125fe650c8101568c9731864"
```

Comment:
Should we have it in example file?

Reply:
Ooopps 👯 , I removed the token from the service

Reply:
Removed from the examples